<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gB_ObslugaSieci">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Obsługa sieci</string>
    </property>
    <widget class="QGroupBox" name="gB_TworzenieSieci">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Tworzenie sieci</string>
     </property>
     <widget class="QRadioButton" name="rB_WczytajSiecZPliku">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wczytaj istniejącą sieć z pliku</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rB_StworzSiecRecznie">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Stwórz nową sieć ręcznie</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wybierz sposób tworzenia sieci:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pB_UsunSiec">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuń zbudowaną sieć</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pB_BudujSiec">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wczytaj/Buduj sieć</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pB_SaveNetwork">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz sieć</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_input1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_input2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_input3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_output">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>441</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gB_teachingParameters">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Parametry uczenia</string>
    </property>
    <widget class="QLineEdit" name="lE_beta">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Współczynnik β: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Współczynnik uczenia η:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_eta">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Współczynnik momentum:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_momentum">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Próg błędu sieci:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_networkErrorThreshold">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.001</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pB_learningExamples">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wczytaj zbiór uczący</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Maksymalna liczba cykli:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lE_maxTeacyingCycleNumber">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>100000</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
